t/op/reset.t

```diff
@@ -175,20 +175,20 @@ pass("no crash when current package is freed");
 undef $/;
 my $prog = <DATA>;
 
-SKIP:
-{
-    eval {require threads; 1} or
-	skip "No threads", 4;
-    foreach my $eight ('/', '?') {
-	foreach my $nine ('/', '?') {
-	    my $copy = $prog;
-	    $copy =~ s/8/$eight/gm;
-	    $copy =~ s/9/$nine/gm;
-	    fresh_perl_is($copy, "pass", {},
-			  "first pattern $eight$eight, second $nine$nine");
-	}
-    }
-}
+#SKIP:
```

Comment:
what's your plan here? come later and fix it or remove it?

Reply:
> what's your plan here? come later and fix it or remove it?

It's on my list for tickets to be created for individual files.

Reply:
See https://github.com/atoomic/perl/issues/93